libraries/PPP/src/PPP.h

```diff
@@ -41,6 +41,9 @@ class PPPClass : public NetworkInterface {
   // Modem DCE APIs
   int RSSI() const;
   int BER() const;
+  int getBatteryVoltage() const;
+  int getBatteryLevel() const;
+  int getBatteryStatus() const;
```

Comment:
to conform with the rest of the naming, I will suggest that you drop `get` and have just `batteryVoltage()` etc.

Reply:
also maybe put them at the end of the paragraph

Reply:
OK, makes sense.